layout/theme.liquid

```diff
@@ -192,6 +192,10 @@
 
     {{ 'base.css' | asset_url | stylesheet_tag }}
 
+    {%- if settings.spacing_grid_horizontal == 0 or settings.spacing_grid_vertical == 0 -%}
+      {{ 'grid-collapsed.css' | asset_url | stylesheet_tag }}
+    {%- endif -%}
```

Comment:
You will need to apply this to `password.liquid` as well. 

You can see that multicolumn looks different on the homepage and the password page, and that's because the password page uses a different `layout`.

Homepage: 
<img width="1159" alt="Screen Shot 2022-04-18 at 12 46 57 PM" src="https://user-images.githubusercontent.com/26778637/163842158-dad3ccae-7df1-4b18-91de-dde3651d2f98.png">

Password: Grid border not collapsing
<img width="1131" alt="Screen Shot 2022-04-18 at 12 47 24 PM" src="https://user-images.githubusercontent.com/26778637/163842219-2dfbc6e8-f6f4-45e7-998b-8ebebfee4c95.png">

Reply:
This has been added.